Assets/_Scripts/3Managers/TimeTicker.cs

```diff
@@ -10,6 +10,7 @@ namespace Assets._Scripts._3Managers
     public class TimeTicker : MonoBehaviour
     {
         public static event EventHandler<int> OnTick;
+        public static readonly int START_OF_THE_GAME = 0;
```

Comment:
I don't see the point of this. Would this value ever change?

Reply:
It's for code readability.